packages/cli/src/cmds/navie/projectInfo.ts

```diff
@@ -13,20 +26,23 @@ export default async function collectProjectInfos(): Promise<ProjectInfo.Project
     delete (stats as Record<string, unknown>).classes;
     delete (stats as Record<string, unknown>).path;
 
-    projectInfoByPath[directory] = { appmapConfig: undefined, appmapStats: stats };
+    projectInfo(directory).appmapStats = stats;
   });
 
   appmapDirectories.forEach((dir) => {
-    projectInfoByPath[dir.directory].appmapConfig = {
+    projectInfo(dir.directory).appmapConfig = {
       name: dir.appmapConfig.name,
       language: dir.appmapConfig.language ?? 'unknown',
       appmap_dir: dir.appmapConfig.appmap_dir ?? 'tmp/appmap',
       packages: dir.appmapConfig.packages,
     };
   });
 
-  // It's good when results are stable.
-  return Object.values(projectInfoByPath).sort((a, b) =>
-    (a.appmapConfig?.name || '').localeCompare(b.appmapConfig?.name || '')
-  );
+  const { projectDirectories } = configuration();
```

Comment:
Be sure and populate the code editor even if the appmap directories is empty (in fact, in that case it's especially important!).